src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -344,7 +344,8 @@ bool AutoDownloadRule::matchesSmartEpisodeFilter(const QString& articleTitle) co
     // See if this episode has been downloaded before
     const bool previouslyMatched = m_dataPtr->previouslyMatchedEpisodes.contains(episodeStr);
     const bool isRepack = articleTitle.contains("REPACK", Qt::CaseInsensitive) || articleTitle.contains("PROPER", Qt::CaseInsensitive);
-    if (previouslyMatched && !isRepack)
+    //if we've already downloaded this, and it's not a repack or repack downloading is disabled, don't download it
```

Comment:
Please fix comment (add space after `//`) or just remove it (I think the expression below isn't so hard to understand it without this comment).